### DISCUSSION TITLE
Upgrade 2.3 - Log files not accessible

### DISCUSSION THREAD

**Discussion by eugenegordeiev** [Kong/kong#7228] [Help]
After upgrading our Kong version to 2.3.2-alpine, Kong logs in /usr/local/kong/logs are not accessible for reading.  They are now streamed to /dev/stdout and /dev/stderr.

We are using filebeat sidecar container to read the logs from a mounted path which points to /usr/local/kong/logs and now it is not working.

I like the idea that we can access the logs right from stdout but we also need ability to read from the files. 

What is the best solution?

**Comment by eugenegordeiev** [comment]
I was able to work around this by changing KONG_PROXY_ACCESS_LOG to /tmp and change the mountPath.  Now filebeat is able to access the logs.

Please suggest if there is a better way to do this?

**Comment by dndx** [comment]
Can you try to change the permissions on `/usr/local/kong/logs` so that filebeat will be able to access them? This does not sounds like a Kong specific issue to me.
